modules/apps/27-interchain-accounts/module.go

```diff
@@ -56,7 +56,7 @@ func (AppModuleBasic) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 
 // ValidateGenesis performs genesis state validation for the IBC interchain acounts module
 func (AppModuleBasic) ValidateGenesis(cdc codec.JSONCodec, config client.TxEncodingConfig, bz json.RawMessage) error {
-	return nil
+	return nil // TODO: https://github.com/cosmos/ibc-go/issues/535
```

Comment:
I figured implementing `ValidateGenesis` should be kept to it's own PR entirely. Wanted to keep this one pretty concise.